resources/email.py

```diff
@@ -32,3 +31,19 @@ def send_reset_password_msg(user):
         msg.html = render_template('emails/reset_msg.html', user=user, token=token)
 
         current_app.mail.send(msg)
+
+        return {"message": "Message sent"}
+
+class InviteEmail(Resource):
+    parser = reqparse.RequestParser()
+    parser.add_argument('email', type=str, required=True, help="This field is not required.")
+
+    def post(self):
```

Comment:
Do you think we should add a test for this?

Reply:
Yes, it should. I can add that once we determine whether the InviteEmail endpoint is needed